x-pack/plugins/cloud_security_posture/public/pages/findings/layout/findings_layout.tsx

```diff
@@ -31,3 +43,72 @@ export const PageTitle: React.FC = ({ children }) => (
 );
 
 export const PageTitleText = ({ title }: { title: React.ReactNode }) => <h2>{title}</h2>;
+
+export const getExpandColumn = <T extends unknown>({
+  onClick,
+}: {
+  onClick(item: T): void;
+}): EuiTableActionsColumnType<T> => ({
+  width: '40px',
+  actions: [
+    {
+      name: 'Expand',
```

Comment:
should be localized

Reply:
https://github.com/elastic/kibana/pull/131422/files#diff-50fc5aaff1c30ab20ada9f3c976d0f05e8150f95e8d660154974db6eae5271beR56-R59